lobby/src/main/java/org/triplea/lobby/server/api/key/ApiKeyGeneratorFactory.java

```diff
@@ -13,11 +14,13 @@
 public final class ApiKeyGeneratorFactory {
 
   /** Creates a new key generator instance that can generate a new API key for a given player. */
-  public static Function<PlayerName, ApiKey> newApiKeyGenerator(final Jdbi jdbi) {
-    return ApiKeyGenerator.builder()
-        .keyMaker(ApiKeyGenerator.createKeyMaker())
-        .apiKeyDao(jdbi.onDemand(ApiKeyDao.class))
-        .userDao(jdbi.onDemand(UserJdbiDao.class))
-        .build();
+  public static BiFunction<PlayerName, InetAddress, ApiKey> newApiKeyGenerator(final Jdbi jdbi) {
+    final ApiKeyGenerator keyGenerator =
+        ApiKeyGenerator.builder()
+            .keyMaker(ApiKeyGenerator.createKeyMaker())
+            .apiKeyDao(jdbi.onDemand(ApiKeyDao.class))
+            .userDao(jdbi.onDemand(UserJdbiDao.class))
+            .build();
+    return (name, ip) -> keyGenerator.apply(name);
```

Comment:
Why is this parameter being added if it's ignored anyways?

Reply:
From the overview:

> Future looking update to send IP data to Api key generation. Future
updates will store IP address with API-keys.

Reply:
Added a TODO here to note that this will be updated.